sky/backends/wheel_utils.py

```diff
@@ -37,13 +37,13 @@ def cleanup_wheels_dir(wheel_dir: pathlib.Path,
 
 
 def _get_latest_built_wheel() -> pathlib.Path:
+    wheel_name = (f'{_PACKAGE_WHEEL_NAME}-'
+                  f'{version.parse(sky.__version__)}-*.whl')
     try:
-        latest_wheel = max(WHEEL_DIR.glob(
-            f'{_PACKAGE_WHEEL_NAME}-'
-            f'{common_utils.normalize_version(sky.__version__)}-*.whl'),
-                           key=os.path.getctime)
+        latest_wheel = max(WHEEL_DIR.glob(wheel_name), key=os.path.getctime)
     except ValueError:
-        raise FileNotFoundError('Could not find built Sky wheels.') from None
+        raise FileNotFoundError(
+            f'Could not find built SkyPilot wheels: {wheel_name}') from None
```

Comment:
nit: f'Could not find built SkyPilot wheels {wheel_name} under {WHEEL_DIR}'?

Reply:
Fixed. Thanks!